mbedtls/Cargo.toml

```diff
@@ -31,6 +31,7 @@ bit-vec = { version = "0.5", optional = true }
 block-modes = { version = "0.3", optional = true }
 rc2 = { version = "0.3", optional = true }
 tokio = { version = "0.3", optional = true }
+proc-macro2 = "=1.0.43"
```

Comment:
I see that current master has `proc-macro2` versions 1.0.24 and 0.4.30 (see Cargo.lock in master), perhaps we could use one of these instead, if it helps?

Reply:
Yes, actually I could reverse the change of:
`proc-macro2 = "=1.0.43"` and `core_io = { version = "0.1.20210325"`
They are used for updateing lock file to resolve deps problem.
But I think it's better to retain them here to give more context info.
Let me update them to sync with master

Reply:
Sure, as long as we don't forget. As per core_io, it's fine, it's removed by in #211 later on.